catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -39,15 +39,15 @@
 
     <feature name="catalog-core" install="manual" version="${project.version}"
              description="Catalog Core feature containing the API, third party bundles necessary to run ddf-core.">
-        <feature>catalog-core-api</feature>
+        <feature prerequisite="true">catalog-core-api</feature>
         <bundle>mvn:ddf.catalog.core/catalog-core-commons/${project.version}</bundle>
-        <bundle>mvn:ddf.catalog.core/catalog-core-camelcomponent/${project.version}</bundle>
         <bundle>mvn:ddf.measure/measure-api/${project.version}</bundle>
         <bundle>mvn:org.codice.thirdparty/picocontainer/1.2_1</bundle>
         <bundle>mvn:org.codice.thirdparty/vecmath/1.3.2_1</bundle>
         <!-- for GeoTools -->
         <bundle>mvn:org.codice.thirdparty/geotools-suite/${org.geotools.bundle.version}</bundle>
         <bundle>mvn:org.codice.thirdparty/jts/${jts.bundle.version}</bundle>
+        <bundle dependency="true">mvn:commons-collections/commons-collections/3.2.1</bundle>
```

Comment:
this bundle was added because the federationstrategy required it.  the `dependency` attribute tells the resolver to only install this bundle if it doesn't already exist, otherwise it would refresh this bundle (I think - it may be completely ignored if you don't of the OBR resolver installed - which we don't).